README.md

```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
+**Note:**
+The phpDocumentor team is currently working on a new phpDocumentor generation. Which will have full support for the latest features in PHP, so you can take advantage of all native type hints. v3 is still under heavy development
+and is not stable enough to use in a daily development flow. 
 
-*Please note* that phpDocumentor 3 is still under heavy development. We aim to add all features needed to have full support
-for php 7+. But at this moment that is not the case.
+### nightly builds
+
+Phpdocumentor doesn't have a nightly release. But during each pipeline, a [phar] artifact is built. If you want 
+to test the bleeding edge version of phpDocumentor have a look in the [actions](https://github.com/phpDocumentor/phpDocumentor/actions?query=workflow%3A%22Qa+workflow%22+is%3Asuccess) section of this repository.
+Each successful QA workflow has a download at the right upper corner.
 
-Requirements
+Installation
 ------------
+phpDocumentor requires php 7.2 or higher to run. However, code of lower php versions can be analyzed.
 
-phpDocumentor requires the following:
+All templates provided with phpDocumentor so have support for Class diagrams based on the read code base. This will
+require Graphviz to be installed on the machine running phpDocumentor. Graphviz is optional, and warnings about missing
+Graphviz can be ignored. However, your documentation will contain some dead links in this case.  
 
-* PHP 7.1 or higher
-* ext/iconv, http://php.net/manual/en/book.iconv.php (is enabled by default since PHP 5.0.0)
-* ext/intl, http://php.net/manual/en/book.intl.php
-* Graphviz (optional, used for generating Class diagrams)
+There are 3 ways to install phpDocumentor:
 
-**Note:**
-If you do not want to install the Graphviz dependency you are encouraged to generate your own template and make sure
-that it does not contain anything related to `Graph`.
-An easier solution might be to edit `data/templates/responsive/template.xml` file and remove every line
-containing the word `Graph` but this will be undone with every upgrade of phpDocumentor.
+1. Using phive (recommended)
+2. Using the PHAR (manual install)
+3. Via [Docker](https://hub.docker.com/r/phpdoc/phpdoc/)
+4. Via [Composer](https://getcomposer.org)
 
-Please see the documentation about creating your own templates for more information.
+### Using Phive
 
-Installation
-------------
+`$ phive install --force-accept-unsigned phpDocumentor`
 
-There are 3 ways to install phpDocumentor:
+For more information about phive have a look at their [website](https://phar.io/) 
+now you have phpDocumentor installed and it can be executed directly. 
 
-1. Using the PHAR
-2. Via [Docker](https://hub.docker.com/r/phpdoc/phpdoc/)
-3. Via [Composer](https://getcomposer.org)
+`php tools/phpDocumentor`
 
 ### Using the PHAR
 
 1. Download the phar file from https://github.com/phpDocumentor/phpDocumentor/releases
-2. ???
-3. Profit!
+2. Profit!
```

Comment:
I'd drop the old "profit" meme, and instead show the explicit syntax to run the phar.